src/pages/explore/explore.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import "./explore.css";
+import { VideoCard } from "../../shared/components/videoCard/VideoCard";
+import { Filter } from "./components/filter/Filter";
```

Comment:
index.js can be created here

Reply:
I will do the refactoring of code that time I will make these changes .